core/camel-endpointdsl/src/main/java/org/apache/camel/builder/endpoint/dsl/RestSwaggerEndpointBuilderFactory.java

```diff
@@ -120,7 +120,7 @@ default AdvancedRestSwaggerEndpointBuilder synchronous(
      * tries to load swagger.json resource from the classpath. Note that the
      * host defined on the component and endpoint of this Component should
      * contain the scheme, hostname and optionally the port in the URI syntax
-     * (i.e. http://api.example.com:8080). Overrides component configuration.
+     * (i.e. \http://api.example.com:8080). Overrides component configuration.
```

Comment:
I don't think we need to escape URLs in JavaDoc, this should be taken care of by the latest changes in the `camel-package-maven-plugin`
```suggestion
     * (i.e. http://api.example.com:8080). Overrides component configuration.
```